.ts_pre_commit_config.yaml

```diff
@@ -1,9 +1,10 @@
-black: true
+black: false
 check-xml: true
 check-yaml: true
-clang-format: true
-flake8: true
-format-xmllint: true
-isort: true
+clang-format: false
+flake8: false
+format-xmllint: false
+isort: false
 mypy: true
 ruff: true
+towncrier: true
```

Comment:
The `false` values of `black`, `clang-format`, and others are intensional or not?

Reply:
Yes, as I explained to you it's either `ruff` or `black`, `flake8` and `isort`. It is `ruff` now so the others are not necessary anymore.